index.html

```diff
@@ -0,0 +1,65 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <nav>
+            <h3 class="logo">My Logo</h3>
+        </nav>
+        
+        <nav>
+            <a href="#">
+            <img src="image/hamburger.png" alt="Hamburger" class="hamburger">
+        </a>
+        </nav>
+       </header>
+
+       <section class="headline">
+        <h1 class="primary">Hey There. <br>I'm Ezra.</h1>
+       
+        <h3 class="supportive">I'm a Software Developer</h3>
+
+        <p class="paragraph">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</p>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** To match the Figma design, I think it would be better if you use the same size like the one used at your template design. i.e ,`font-size: 16px`